composer/helpers/v2/src/Updater.php

```diff
@@ -88,7 +88,8 @@ public static function update(array $args): array
             ->setExecuteOperations(true)
             ->setDumpAutoloader(false)
             ->setPlatformRequirementFilter(PlatformRequirementFilterFactory::fromBoolOrList(false))
-            ->setAudit(false);
+            ->setAudit(false)
+            ->setMinimalUpdate(true);
```

Comment:
Hi 👋🏾 thanks for the contribution!

It looks like this is the only change on this PR aside from the Composer 2.6.5 --> 2.7.2 upgrade?

Since the default is `true` I'd like to deploy this PR separate from the upgrade so we can see if it does anything